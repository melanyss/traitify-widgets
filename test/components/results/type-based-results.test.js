import Component from "components/results/type-based-results";
import ComponentHandler from "support/component-handler";

jest.mock("components/results/type-based-results/personality-base", () => (() => (<div className="mock">Personality Base</div>)));
jest.mock("components/results/type-based-results/personality-details", () => (() => (<div className="mock">Personality Details</div>)));
jest.mock("components/results/type-based-results/personality-traits", () => (() => (<div className="mock">Personality Traits</div>)));
jest.mock("components/results/type-based-results/personality-types", () => (() => (<div className="mock">Personality Types</div>)));
jest.mock("lib/with-traitify");

describe("TypeBasedResults", () => {
  it("renders results", () => {
    const component = new ComponentHandler(<Component />);

    expect(component.tree).toMatchSnapshot();
  });
});
