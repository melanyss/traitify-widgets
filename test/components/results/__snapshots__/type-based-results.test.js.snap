// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypeBasedResults renders results 1`] = `
<section>
  <div
    className="mock"
  >
    Personality Base
  </div>
  <div
    className="mock"
  >
    Personality Types
  </div>
  <div
    className="mock"
  >
    Personality Traits
  </div>
  <div
    className="mock"
  >
    Personality Details
  </div>
</section>
`;
